[2022-05-26 18:57:01,797] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: Etl.create_employees_table manual__2022-05-26T18:57:01.359861+00:00 [queued]>
[2022-05-26 18:57:01,805] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: Etl.create_employees_table manual__2022-05-26T18:57:01.359861+00:00 [queued]>
[2022-05-26 18:57:01,805] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 18:57:01,805] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-26 18:57:01,805] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 18:57:01,815] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): create_employees_table> on 2022-05-26 18:57:01.359861+00:00
[2022-05-26 18:57:01,818] {standard_task_runner.py:52} INFO - Started process 9284 to run task
[2022-05-26 18:57:01,821] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Etl', 'create_employees_table', 'manual__2022-05-26T18:57:01.359861+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmp1rkb1ekw', '--error-file', '/tmp/tmpaowc43x2']
[2022-05-26 18:57:01,821] {standard_task_runner.py:80} INFO - Job 26: Subtask create_employees_table
[2022-05-26 18:57:01,861] {task_command.py:370} INFO - Running <TaskInstance: Etl.create_employees_table manual__2022-05-26T18:57:01.359861+00:00 [running]> on host b7bcf2b221fa
[2022-05-26 18:57:01,921] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Etl
AIRFLOW_CTX_TASK_ID=create_employees_table
AIRFLOW_CTX_EXECUTION_DATE=2022-05-26T18:57:01.359861+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-26T18:57:01.359861+00:00
[2022-05-26 18:57:01,930] {base.py:68} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2022-05-26 18:57:01,933] {dbapi.py:208} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );, parameters: None
[2022-05-26 18:57:01,933] {postgres.py:94} INFO - NOTICE:  relation "employees" already exists, skipping

[2022-05-26 18:57:01,953] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=Etl, task_id=create_employees_table, execution_date=20220526T185701, start_date=20220526T185701, end_date=20220526T185701
[2022-05-26 18:57:01,992] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-26 18:57:02,024] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
